### PR TITLE
Add code background to highlight front page "Open Source" section

### PR DESCRIPTION
**Pull request by Holzhaus**
2020-06-10-124922_2637x1585_scrot

### CODE REVIEW

**Comment by Holzhaus** [comment]
2020-06-10-132436_2655x1385_scrot

**Comment by katsar0v** [comment]
I like it! Definitely better (imo) then the current state. The background can be discussed, maybe some blur or other photo, but that's a small details that does not have to be discussed a lot

File: static/css/custom.css [hunk 1]
```diff
@@ -416,15 +416,15 @@ h6 a.headline-link {
...
-.splash-text h3,
-.splash-text h4,
-.splash-text h5,
-.splash-text h6 {
+.splash .splash-text h1,
+.splash .splash-text h2,
+.splash .splash-text h3,
+.splash .splash-text h4,
+.splash .splash-text h5,
+.splash .splash-text h6 {
```
Comment (xeruf) [inline static/css/custom.css:427]: do we need a selector for every heading if there's only a h2? Or did I miss something?

**Review by xeruf — APPROVED** [review]
Only had a glance over the code, but the change LGTM :)

Looking back at the current website, yes it can definitely use some more colors/images!

**Comment by ronso0** [comment]
This looks great!
Besides making the page more lively, the code bg also serves as section separator.

LGTM

File: static/css/custom.css (hunk 1 above)
Comment (ronso0) [reply static/css/custom.css:427]: IME it's just convenient to have style available when adding other .splash-texts in the future.

**Comment by Holzhaus** [comment]
I replaced the screenshot for obvious reasons. Feel free to merge now :smile:

File: pages/index.html [hunk 2]
```diff
@@ -79,23 +79,25 @@ <h5 class="docs-header">{% trans "Vinyl Record Control" %}</h5>
...
-  <h2><a class="headline-link" href="#opensource">{% trans "Free &amp; Open Source" %}</a></h2>
-  <p>
-    {% blocktrans %}Mixxx is free open-source software and entirely community-driven. There is no company behind Mixxx - the development is shouldered by passionate DJs and programmers that dedicate their free time on working on their favorite DJ software. Mixxx is and always will be free! See dozens more features or download Mixxx and start DJing now!{% endblocktrans %}
-  </p>
-  <div class="grid-container quarters">
+<div id="opensource" class="docs-section cover-bg" style="background-image: url({% static '/static/images/cover/code-bg.jpg' %});">
+  <div class="container splash-text">
+    <h2><a class="headline-link" href="#opensource">{% trans "Free &amp; Open Source" %}</a></h2>
+    <p>
+      {% blocktrans %}Mixxx is free open-source software and entirely community-driven. There is no company behind Mixxx - the development is shouldered by passionate DJs and programmers that dedicate their free time on working on their favorite DJ software. Mixxx is and always will be free! See dozens more features or download Mixxx and start DJing now!{% endblocktrans %}
```
Comment (Be-ing) [inline pages/index.html]:
```suggestion
      {% blocktrans %}Mixxx is free open-source software and entirely community-driven. There is no company behind Mixxx &mdash; the development is shouldered by passionate DJs and programmers that dedicate their free time to working on their favorite DJ software. Mixxx is and always will be free! See dozens more features or download Mixxx and start DJing now!{% endblocktrans %}
```

**Pull request merged**